### DISCUSSION TITLE
[Announcement] Final exam

### DISCUSSION THREAD

**Discussion by yeonhee-ryou** [prosyslab-classroom/cs348-information-security#446] [Announcements]
* Time: June 12. (Mon) 9:00 - 10:30 (90 mins)
* Place: N1 117 (same as the classroom)
* Open book but no digital devices allowed
* Write your answers in Korean if you are a native Korean student; English otherwise
* No question is allowed during the exam
* No partial points will be given

**Comment by hoseong02** [comment]
Hello. I'm Hoseong Yang, taking CS348.

I have a question about final exam.

Are all native Korean students supposed to write there answers in only Korean?
Is it allowed to write it in English?

Thank you.

**Comment by KihongHeo** [comment]
Yes. Write in Korean.

Is there any reason why you prefer English even though you are a "native" Korean speaker?

Here is why we require this rule:
- Sometimes it is nontrivial to understand some answers from students. Hard to tell whether it is due to language problem or the student gives a wrong answer. I don't want an English test which is undesired.
- You should be able to clearly explain what you understand in your native language (as well as English for sure).

**Comment by hoseong02** [comment]
I've been studying with the lecture notes, and all terms are in English.
I'd like to ask that if we are allowed to write answer with terms in English.
Thank you.

**Comment by KihongHeo** [comment]
That is okay for sure.

우리말 용어가 잘 떠오르지 않는 전문 용어인 경우 우리 시험에서는 영어로 써도 무방합니다. 글짓기 시험이 아니기 때문입니다. (예: 해당 ciphertext 를 decrypt 하여 어쩌고..., hash 값을 계산하고 어쩌고, ... 등)

허나 꼭 명심하세요. 나중에 각잡고 쓰는 공식 문서 (예: 지원서, 보고서, 제안서 등) 에는 그런식으로 쓰면 안됩니다. 본인의 수준낮음과 성의 없음을 증명하는 꼴입니다.

**Comment by hoseong02** [comment]
I will keep that in my mind.
Thank you for your kind answer.
